docs/index.rst

```diff
@@ -0,0 +1,103 @@
+.. toctree::
+   :hidden:
+
+   self
+
+Hello, I'm Elodie
```

Comment:
Just copied over the intro paragraph from the readme. I can change it to something else, if you'd like.